docs/source/en/guides/cli.md

```diff
@@ -0,0 +1,445 @@
+Token is valid (permission: write).
+Your token has been saved in your configured git credential helpers (store).
+Your token has been saved to /home/wauplin/.cache/huggingface/token
+Login successful
+```
+
+## huggingface-cli whoami
+
+If you want to know if you are logged in, you can use `huggingface-cli whoami`. This command doesn't have any option and will simply print your username and the organizations you are part of on the Hub:
+
+```bash
+huggingface-cli whoami                                                                     
+Wauplin
+orgs:  huggingface,eu-test,OAuthTesters,hf-accelerate,HFSmolCluster
+```
+
+If you are not logged in, an error message will be printed.
+
+## huggingface-cli logout
+
+This commands logs you out. In practice, it will delete the token saved on your machine.
+
+<Tip warning={true}>
+
+This command will not log you out if you are logged in using the `HF_TOKEN` environment variable (see [reference](../package_reference/environment_variables#hftoken)). If that is the case, you must unset the environment variable in your machine configuration.
+
+</Tip>
+
+## huggingface-cli download
+
+
+Use the `huggingface-cli download` command from the terminal to directly download files from the Hub. Internally, it uses the same [`hf_hub_download`] and [`snapshot_download`] helpers described in the [Download](./download) guide and prints the returned path to the terminal. In the examples below, we will walk through the most common use cases. For a full list of available options, you can run:
+
+```bash
+huggingface-cli download --help
+```
+
+### Download a single file
+
+To download a single file from a repo, simply provide the repo_id and filename as follow:
+
+```bash
+>>> huggingface-cli download gpt2 config.json
+downloading https://huggingface.co/gpt2/resolve/main/config.json to /home/wauplin/.cache/huggingface/hub/tmpwrq8dm5o
+(…)ingface.co/gpt2/resolve/main/config.json: 100%|██████████████████████████████████| 665/665 [00:00<00:00, 2.49MB/s]
+/home/wauplin/.cache/huggingface/hub/models--gpt2/snapshots/11c5a3d5811f50298f278a704980280950aedb10/config.json
+```
+
+The command will always print on the last line the path to the file on your local machine.
+
+### Download an entire repository
+
+In some cases, you just want to download all the files from a repository. This can be done by not specifying any filename:
+
+```bash
+>>> huggingface-cli download HuggingFaceH4/zephyr-7b-beta
+Fetching 23 files:   0%|                                                | 0/23 [00:00<?, ?it/s]
+...
+...
+/home/wauplin/.cache/huggingface/hub/models--HuggingFaceH4--zephyr-7b-beta/snapshots/3bac358730f8806e5c3dc7c7e19eb36e045bf720
+```
+
+### Download multiple files
+
+You can also download a subset of the files from a repository in a single command. This can be done in two ways. If you already have a precise list of the files you want to download, you can simply provide them as a sequence:
```

Comment:
```suggestion
You can also download a subset of the files from a repository with a single command. This can be done in two ways. If you already have a precise list of the files you want to download, you can simply provide them sequentially:
```